Formula/sbt.rb

```diff
@@ -20,11 +20,24 @@ def install
       s.gsub! "/etc/sbt/sbtopts", "#{etc}/sbtopts"
     end
 
-    inreplace "bin/sbt-launch-lib.bash", "${sbt_home}/bin/sbt-launch.jar", "#{libexec}/sbt-launch.jar"
+    inreplace "bin/sbt-launch-lib.bash" do |s|
+      s.gsub! "$(dirname \"$(realpath \"$0\")\")", libexec
+      s.gsub! "$(dirname \"$sbt_bin_dir\")", libexec
+      s.gsub! "${sbt_home}/bin/sbt-launch.jar", "#{libexec}/sbt-launch.jar"
+      ## This is required to pass the test
+      if s.include?("[[ \"$java_version\" > \"8\" ]]")
+        s.gsub! "[[ \"$java_version\" > \"8\" ]]", "[[ \"$java_version\" == \"9\" ]]"
+      end
+    end
 
-    libexec.install "bin/sbt", "bin/sbt-launch-lib.bash", "bin/sbt-launch.jar"
+    libexec.install "bin/sbt", "bin/sbt-launch-lib.bash"
+    libexec.install Dir["bin/*.jar"]
     etc.install "conf/sbtopts"
 
+    if File.directory?("lib")
```

Comment:
So it specifically is for stable only? In that case you can make it `if build.stable?` and add a comment that the if should be removed when devel is > 1.0.0-M4.